src/bin/rgb/command.rs

```diff
@@ -667,6 +680,63 @@ impl Command {
                 runtime.accept_transfer(transfer, force)?;
                 eprintln!("Transfer accepted into the stash");
             }
+            Command::SetHost { method, psbt_file } => {
+                let psbt_data = fs::read(&psbt_file)?;
+                let mut psbt = Psbt::deserialize(&psbt_data)?;
+                let mut psbt_modified = false;
+                match method {
+                    CloseMethod::OpretFirst => {
+                        if let Some((index, _)) = psbt
+                            .unsigned_tx
+                            .output
+                            .iter_mut()
+                            .enumerate()
+                            .find(|(_, o)| o.script_pubkey.is_op_return())
+                        {
+                            if let Some((_, outp)) = psbt
+                                .outputs
+                                .iter_mut()
+                                .enumerate()
+                                .find(|(i, _)| i == &index)
+                            {
+                                if !outp.is_opret_host() {
+                                    psbt_modified = true;
+                                    outp.set_opret_host().expect("set opret host");
```

Comment:
I usually do 
```suggestion
                                    outp.set_opret_host().ok();
```
in such cases

Reply:
done